frontend/src/static/js/components/webstatus-feature-wpt-progress-chart-panel.ts

```diff
@@ -26,20 +26,21 @@ import {
   STABLE_CHANNEL,
   WPTRunMetric,
 } from '../api/client.js';
-import {
-  FetchFunctionConfig,
-  WebstatusLineChartPanel,
-} from './webstatus-line-chart-panel.js';
+import {FetchFunctionConfig} from './webstatus-line-chart-panel.js';
+import {WebstatusLineChartTabbedPanel} from './webstatus-line-chart-tabbed-panel.js';
 
 @customElement('webstatus-feature-wpt-progress-chart-panel')
-export class WebstatusFeatureWPTProgressChartPanel extends WebstatusLineChartPanel {
-  readonly featureSupportBrowsers: BrowsersParameter[] = [
-    'chrome',
-    'firefox',
-    'safari',
-    'edge',
+export class WebstatusFeatureWPTProgressChartPanel extends WebstatusLineChartTabbedPanel {
+  readonly browsersByView: BrowsersParameter[][] = [
+    ['chrome', 'firefox', 'safari', 'edge'],
+    ['chrome_android', 'firefox_android', 'safari_ios'],
   ];
+
+  @state()
```

Comment:
Does this need to be reactive with `@state`?

Reply:
Fixed!

Reply:
whoops, I thought I squashed all of these designated as "state" - sorry about that!